pkg/gofr/datasource/pubsub/eventhub/eventhub_test.go

```diff
@@ -22,20 +22,20 @@ func TestConnect(t *testing.T) {
 
 	mockLogger := NewMockLogger(ctrl)
 
-	mockLogger.EXPECT().Debug("azure eventhub connection started using connection string")
-	mockLogger.EXPECT().Debug("azure eventhub producer client setup success")
-	mockLogger.EXPECT().Debug("azure eventhub container client setup success")
-	mockLogger.EXPECT().Debug("azure eventhub blobstore client setup success")
-	mockLogger.EXPECT().Debug("azure eventhub consumer client setup success")
-	mockLogger.EXPECT().Debug("azure eventhub processor setup success")
-	mockLogger.EXPECT().Debug("azure eventhub processor running successfully").AnyTimes()
+	mockLogger.EXPECT().Debug("azure Event Hub connection started using connection string")
```

Comment:
```suggestion
	mockLogger.EXPECT().Debug("Azure Event Hub connection started using connection string")
```

Azure is a brand

Maybe another search and replace to do.

But then here, I wonder whether having Azure Event Hub make sense, I mean Event Hub could be enough in logs and errors.

I mean we are not using AWS S3 or Amazon S3 for S3, or Google Pubsub

Reply:
Not exactly

https://go.dev/wiki/CodeReviewComments#error-strings
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.

Reply:
Sure removed `azure` from logs.